src/converse-core.js

```diff
@@ -280,6 +281,10 @@
             }
         }
 
+        if(this.time_format !== 'h' && this.time_format !== 'H') {
+            throw new Error("Config Error: wrong time format configuration");
+        }
+
```

Comment:
This must be removed since we don't want to restrict the values the user can enter as the format.